app/src/main/java/com/gathering/android/profile/editProfile/ProfileException.kt

```diff
@@ -0,0 +1,6 @@
+package com.gathering.android.profile.editProfile
+
+sealed class ProfileException : Throwable() {
+    data object ServerNotRespondingException : ProfileException()
```

Comment:
Server not responding can be a general error for all API calls not only this one.